### PR TITLE
Refactoring one

### PR DESCRIPTION
**Pull request by lisa-von-mond**
* a lot of code cleaning
* logic of counters, levels and pop ups refactoring

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,19 @@
...
+This is a capstone-project of my web development bootcamp at https://github.com/neuefische
 
-The game field consists of a base planet and further planet scopes, subgroups of lower order planets. A lower order planet can only be reached from the corresponding higher order planet and vice versa.
+## ABOUT THIS GAME
 
-### How to navigate
-
-Jumping to lower order planets (further from base planet) is possible with help of the command FAR. Since there are often more than one possibility to hop further, there is the focus indicator. With help of the commands LEFT and RIGHT, the focus can be changed in clockwise or anti-clockwise direction.
-Hopping to higher order planets - closer to base station - works by the command CLOSE.
+lush3000 is a game for kids from 9-99 years.
```
Comment (Carolin-Weitkamp) [inline README.md:9]: did you rename it? 
i liked bloomcraft a lot - but lush 3000 is also a good choice ;)

File: README.md [hunk 2]
```diff
@@ -1,30 +1,19 @@
...
-2 CLOSE will be resolved as CLOSE CLOSE
-3 FAR RIGHT will be resolved as FAR RIGHT FAR RIGHT FAR RIGHT 
-For completing a level successfully, you will have to navigate to the planet with the seeds - passing this planet will let you automatically pick the seeds - and then navigate to your destination. 
+Every level plays in a different galaxy. Each galaxy consists of a base planet and further planet scopes, subgroups of lower order planets. Navigate your spaceship based on logic principles. Choose your commands efficiently. Using more commands than allowed or to fire commands, that can not be executed, means level fail.
 
 ## CONTRIBUTING
 
-For everyone reading this: If you are interested in creating a level, feel free to contact me!
-
+information will follow here
```
Comment (Carolin-Weitkamp) [inline README.md:19]: nice - i really like the idea that the spaceships transport seed ressources

Comment (Carolin-Weitkamp) [reply README.md:19]: all in all its good to understand

**Review by Carolin-Weitkamp — COMMENTED** [review]
i saw that its a lot of detail work - i just had a brief look.
my concentration is not high enough to read everything. 
But what i saw/read looked good to me.

:)

File: README.md (hunk 2 above)
Comment (lisa-von-mond) [reply README.md:19]: thx!

File: README.md (hunk 1 above)
Comment (lisa-von-mond) [reply README.md:9]: I changed it because bloomcraft already exists in other context...

**Pull request merged**